File: Logic Building Problems/Basic Problems/Check Even or Odd/logic3.java
// Java program to
// check for even or odd
class logic3{
    public static boolean isEven(int n){
       if (n == (n >> 1) << 1) 
            return true;
        else 
            return false;
    }

    public static void main(String[] args) {
        int n = 4;
        if (isEven(n) == true)
            System.out.print("true");
        else
            System.out.print("false");
    }
}

File: Logic Building Problems/Basic Problems/Check Even or Odd/logic3.js
// A simple JavaScript program to
// check for even or odd

function isEven(n) {
   
    if (n == (n >> 1) << 1) 
       return true;
    else 
       return false;
}

// Driver code
let n = 4;
if (isEven(n)) {
    console.log("true");
} else {
    console.log("false");
}